### PR TITLE
ImagesTable: update empty-table message

### PR DESCRIPTION
**Pull request by teg**
Update the message now that we support CentOS Stream, Azure+GCP support as well as package selection.

### CODE REVIEW

File: src/SmartComponents/ImagesTable/ImagesTable.js [hunk 1]
```diff
@@ -88,7 +88,8 @@ class ImagesTable extends Component {
                             Create an image
                         </Title>
                         <EmptyStateBody>
-                            Create RHEL OS images for deployment in Amazon Web Services. Images can include
+                            Create OS images for deployment in Amazon Web Services, Azure and
```
Comment (Gundersanne) [inline src/SmartComponents/ImagesTable/ImagesTable.js:91]: Maybe use "Microsoft Azure" here, whenever we use the full name we tend to use that.

Comment (teg) [reply src/SmartComponents/ImagesTable/ImagesTable.js:91]: Ah, I meant to change that. Thanks for catching it!

Comment (teg) [reply src/SmartComponents/ImagesTable/ImagesTable.js:91]: Done.

**Comment by teg** [comment]
Replaced by #109

**Pull request closed without merging**